nifi-extension-bundles/nifi-websocket-bundle/nifi-websocket-services-jetty/src/main/java/org/apache/nifi/websocket/jetty/JettyWebSocketSession.java

```diff
@@ -66,7 +72,15 @@ public InetSocketAddress getLocalAddress() {
 
     @Override
     public boolean isSecure() {
-        return session.isSecure();
+        return secure;
+    }
+
+    private static boolean checkSecure(final Session session) {
```

Comment:
Recommend adjust the method name.
```suggestion
    private static boolean isSessionSecure(final Session session) {
```